README-MLservices.md

```diff
@@ -0,0 +1,169 @@
+
+  # RDS Information
+  rds_allocated_storage = "100"
+  rds_backup_retention  = "0"
+  rds_backup_window     = "03:00-04:00"
+  rds_db_instance_size  = "db.m4.xlarge"
+  rds_db_password       = "mydbpassword"
+  rds_db_username       = "mydbuser"
+  rds_multi_az          = false
+  rds_snapshot          = "" # Set to "final" after the initial deployment
+  rds_subnet_id_1       = "${module.network.rds_subnet_id_1}"
+  rds_subnet_id_2       = "${module.network.rds_subnet_id_2}"
+}
+
+# nld - (Optional) Natural Language Description extractor.
+module "nld" {
+  source = "github.com/graymeta/terraform-aws-platform//modules/ml_services/nld?ref=v0.1.5"
+
+  instance_type          = "m5.large"
+  max_cluster_size       = "2"
+  min_cluster_size       = "1"
+  ml_loadbalancer_output = "${module.ml_network.ml_loadbalancer_output}"
+  services_ecs_cidrs     = ["${module.network.ecs_cidrs}", "${module.network.services_cidrs}"]
+}
+
+output "nld_endpoint" {
+  value = "${module.nld.endpoint}"
+}
+
+# object - (Optional) GrayMeta Object Detection extractor.
+module "object" {
+  source = "github.com/graymeta/terraform-aws-platform//modules/ml_services/object?ref=v0.1.5"
+
+  instance_type          = "m5.large"
+  max_cluster_size       = "2"
+  min_cluster_size       = "1"
+  ml_loadbalancer_output = "${module.ml_network.ml_loadbalancer_output}"
+  services_ecs_cidrs     = ["${module.network.ecs_cidrs}", "${module.network.services_cidrs}"]
+}
+
+output "object_endpoint" {
+  value = "${module.object.endpoint}"
+}
+
+# slates - (Optional) Slates extractor.
+module "slates" {
+  source = "github.com/graymeta/terraform-aws-platform//modules/ml_services/slates?ref=v0.1.5"
+
+  instance_type          = "m5.large"
+  max_cluster_size       = "2"
+  min_cluster_size       = "1"
+  ml_loadbalancer_output = "${module.ml_network.ml_loadbalancer_output}"
+  services_ecs_cidrs     = ["${module.network.ecs_cidrs}", "${module.network.services_cidrs}"]
+}
+
+output "slates_endpoint" {
+  value = "${module.slates.endpoint}"
+}
+
+# vssoccer - (Optional) Graymeta Visual Sports Soccer extractor
+module "vssoccer" {
+  source = "github.com/graymeta/terraform-aws-platform//modules/ml_services/vssoccer?ref=v0.1.5"
+
+  instance_type          = "m5.large"
+  max_cluster_size       = "2"
+  min_cluster_size       = "1"
+  ml_loadbalancer_output = "${module.ml_network.ml_loadbalancer_output}"
+  services_ecs_cidrs     = ["${module.network.ecs_cidrs}", "${module.network.services_cidrs}"]
+}
+
+output "vssoccer_endpoint" {
+  value = "${module.vssoccer.endpoint}"
+}
+```
+
+## To setup the endpoint for the extractors
+#### GrayMeta Facial Recognition extractor
+For the Graymeta Facial Recognigtion extractor you will need to add the following variables to the platform module.
+```
+module "platform" {
+  source = "github.com/graymeta/terraform-aws-platform?ef=v0.1.5"
+  ...
+  # (Optional) Graymeta ML Services
```

Comment:
should this soccer reference be removed since it's discussed below in the `All the other extractors` section?

Reply:
soccer?  Are you asking about the 
```
  # (Optional) Graymeta ML Services
  mlservices_endpoint = "${module.ml_network.endpoint}"
```
So that ml_services_endpoint is what goes into the no_proxy on services and ecs.  Let my try something different here.